.github/workflows/smi-conformace.yml

```diff
@@ -0,0 +1,91 @@
+
+name: smi-conformance
+
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    env:
+      # mesheryctl or helm
+      DEPLOYMENT_MODE: mesheryctl
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      
+      - name: Install mesheryctl
+        run: |
+          /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+          eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+          brew tap layer5io/tap
+          brew install mesheryctl
+          
+      - name: copy default config
+        run: |
+          mkdir -p  ~/.meshery
+          cd  ~/.meshery
+          wget https://raw.githubusercontent.com/layer5io/meshery/37af1e8fea802ac754e83473cb9f63d97d8d850f/mesheryctl/pkg/utils/TestConfig.yaml 
+          mv TestConfig.yaml config.yaml 
+          
+      - name: verify install
+        run: mesheryctl version
+        
+      - name: install go
+        run: sudo apt install golang-go
+        
+      - name: install kind
+        run: GO111MODULE="on" go get sigs.k8s.io/kind@v0.10.0 && kind create cluster
+        
+      - name: export kubeconfig
```

Comment:
Is this for Meshery’s sake or kubectl’s?

Reply:
Kubectl's